website/client/src/components/static/home.vue

```diff
@@ -844,6 +860,14 @@ export default {
       if (this.username.length < 1) return false;
       return !this.usernameValid;
     },
+    passwordValid () {
+      if (this.password.length <= 0) return false;
+      return this.password.length >= MINIMUM_PASSWORD_LENGTH;
+    },
+    passwordInvalid () {
+      if (this.password.length <= 0) return false;
+      return this.password.length < MINIMUM_PASSWORD_LENGTH;
+    },
     passwordConfirmValid () {
       if (this.passwordConfirm.length <= 3) return false;
```

Comment:
Should this be changed to 8 or the whole line deleted? I don't think a length check is needed here because the only important check is that the password confirm value is the same as the password value (and we're already checking length for the password).

Same for passwordConfirmInvalid below.

Reply:
The <= 3 was a previous check that I think was supposed not to show the error until the user has typed at least 3 characters, which I feel like makes sense from an UX perspective but I'm not 100% sure